### PR TITLE
Add migrate control node in tiup-cluster

### PR DESCRIPTION
**Pull request by july2993**
### What is changed, added or deleted? (Required)
Add migrate control node in tiup-cluster

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
fix https://github.com/pingcap/tiup/issues/681

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by july2993** [comment]
@lonng PTAL

File: tiup/tiup-cluster.md [hunk 1]
```diff
@@ -626,3 +626,15 @@ export TIUP_NATIVE_SSH=enable
...
+
+TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，迁移中控机只需要拷贝这个目录到对应目标机器即可。
+
+1. 在原机器 home 目录下运行 `tar czvf tiup.tar.gz .tiup`
+2. 把 `tip.tar.gz` 拷贝到目标机器 home 目录
+3. 在目标机器 home 目录下运行 `tar xzvf tiup.tar.gz`
+4. 添加 `.tiup` 目录到 `PATH` 环境变量
+   * 如在 `~/.bashr` 添加 `export PATH=/home/tidb/.tiup/bin:$PATH` 后运行 `source ~/.bashrc`
+
+为了避免中控机磁盘损坏等异常情况丢失 TiUP 的数据，建议定时备份 `.tiup` 目录。
```
Comment (lonng) [inline tiup/tiup-cluster.md:640]:
```suggestion
> 为了避免中控机磁盘损坏等异常情况丢失 TiUP 的数据，建议定时备份 `.tiup` 目录。
```

File: tiup/tiup-cluster.md [hunk 2]
```diff
@@ -626,3 +626,15 @@ export TIUP_NATIVE_SSH=enable
...
+
+## 迁移中控机与备份
+
+TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，迁移中控机只需要拷贝这个目录到对应目标机器即可。
+
+1. 在原机器 home 目录下运行 `tar czvf tiup.tar.gz .tiup`
+2. 把 `tip.tar.gz` 拷贝到目标机器 home 目录
+3. 在目标机器 home 目录下运行 `tar xzvf tiup.tar.gz`
+4. 添加 `.tiup` 目录到 `PATH` 环境变量
+   * 如在 `~/.bashr` 添加 `export PATH=/home/tidb/.tiup/bin:$PATH` 后运行 `source ~/.bashrc`
```
Comment (lonng) [inline tiup/tiup-cluster.md:638]: 说明一下这个是 bash 改这个脚本，如果其他 shell 需要改不同的文件。

Comment (july2993) [reply tiup/tiup-cluster.md]: https://github.com/pingcap/docs-cn/pull/4272/commits/ed0d00c28ed9acd32293ccfd431db30ce695f0db

**Pull request merged**